### PR TITLE
populate models with admin popularity/population metadata

### LINKED ISSUE #51: Scoring based on the admin area it belongs to [PL-AG05]
Inherit score from enclosing admin area

related issues:
* https://github.com/pelias/admin-lookup/issues/5
* https://github.com/pelias/pelias/issues/46

### PR DESCRIPTION
**Pull request by sevko**
Extract population/popularity values from Quattro polys and inject them into inbound `Document`s coming through `.stream()` using `.setPopulation()` and `.setPopularity()`. Will facilitate results boosting for every record passing through the `admin-lookup`.

Implements #5 .
Fixes pelias/pelias#51

### CODE REVIEW

**Comment by sevko** [comment]
Needs to be tested in dev imports before getting merged.

File: lib/master.js [hunk 1]
```diff
@@ -134,18 +135,33 @@ function createLookup( workers ){
...
+
+    var popularities = [
+      resps.admin1.popularity,
+      resps.admin2.popularity,
+      resps.local_admin.popularity,
+      resps.locality.popularity,
+      resps.neighborhood.popularity
+    ];
+    result.popularity = popularities.reduce( function ( acc, curr ){
+      return ( typeof curr === 'number' ) ?
```
Comment (hkrishna) [inline lib/master.js]: Not sure if this check is sufficient.. Beware of Javascript demons! 

```javascript
  typeof NaN = 'number'
```
how about ```isNaN()``` ?

File: lib/master.js [hunk 2]
```diff
@@ -134,18 +135,33 @@ function createLookup( workers ){
...
+        acc + curr :
+        acc;
+    });
+    console.log( result );
```
Comment (hkrishna) [inline lib/master.js]: is this for debugging purposes? should we delete this line before merging into master?

Comment (sevko) [inline lib/master.js]: Indeed, I accidentally committed this and then deleted locally (but never pushed).

File: lib/master.js (hunk 1 above)
Comment (sevko) [inline lib/master.js]: Better safe than sorry, will add.

**Comment by hkrishna** [comment]
@sevko can you resolve conflicts and make this PR merge-able again? :)

**Comment by hkrishna** [comment]
+1

**Pull request merged**